src/r5py/r5/transport_network.py

```diff
@@ -79,9 +80,18 @@ def __init__(self, osm_pbf, gtfs=[]):
             transport_network.transitLayer = com.conveyal.r5.transit.TransitLayer()
             transport_network.transitLayer.parentNetwork = transport_network
             for gtfs_file in gtfs:
-                gtfs_feed = com.conveyal.gtfs.GTFSFeed.readOnlyTempFileFromGtfs(
+                gtfs_feed = com.conveyal.gtfs.GTFSFeed.writableTempFileFromGtfs(
```

Comment:
For the sake of remembering why I changed this: readOnlyTempFile does not report errors